service.go

```diff
@@ -334,6 +351,9 @@ type Service interface {
 	// String displays the name of the service. The display name if present,
 	// otherwise the name.
 	String() string
+
+	// Status returns the current service status.
```

Comment:
Possibly name the return arguments. It looks like the string is the system assigned status, while the Status is the interpretation of that string.

Reply:
I'm not convinced this should return a string. Can you provide an argument for when you would need this information?

Reply:
At this point it may be safe to remove this. The original idea for the string output was to provide the output of the command incase more detail could be extracted for the consumer.